packages/usa-table/src/usa-table.twig

```diff
@@ -14,24 +26,28 @@
   {% endif %}
 
   <thead>
-    <tr>
-      {% for th in thead %}
-        <th scope="{{ th.scope|default('col') }}">{{ th.title }}</th>
-      {% endfor %}
-    </tr>
-  </thead>
-  <tbody>
-    {% for row in tbody %}
+    {% for i in headerLoopCount %}
       <tr>
-        {% for cell in row %}
-          {% if not cell.scope %}
-            <td>{{ cell.title }}</td>
-          {% else %}
-            <th scope="{{ cell.scope }}">{{ cell.title }}</th>
-          {% endif %}
+        {% for th in thead %}
+          <th scope="{{ th.scope|default('col') }}">{{ th.title }}</th>
         {% endfor %}
       </tr>
     {% endfor %}
```

Comment:
This tests that all `thead` content is visible when there are multiple rows in a sticky header. Wanted to include a test for this because initially only the first row in the `thead` would stick and the other row(s) would move with  the rest of the unsticky content. 

To minimize confusion in the future, I've added a note to the control in [58d0a6a](https://github.com/uswds/uswds/pull/5420/commits/58d0a6afef50b8ddf24e55f0dc3eb2ff57fbd855).

Reply:
If the control for multiple header rows is strictly for sticky header testing — a separate test might be clearer.

Reply:
Good idea! Removed the universal control and created a test pattern for it instead in [1e447af](https://github.com/uswds/uswds/pull/5420/commits/1e447afdeb30cc95a231c6c6831104fd0dcd6403)